content/_releases/24/17.md

```diff
@@ -0,0 +1,86 @@
+* `per_workspace_request_webhooks_per_minute_limit`: controls calls to webhooks per minute at the workspace level.
+
+These quotas enable fine-grained management of webhook usage, ensuring optimal performance and resource utilization.
+
+**Error notification for Quota exceedance**
+
+When the quota is exceeded, the following error message will be displayed:
+```
+{
+  "error": "Request quota exceeded: contract 5b5ed1cf272cf80011ae7b43 (false) | workspace 59d341e9037f7200184a408b (true)",
+  "requestId": "9a315b2c476bd2c6f40a8d739d3f348e"
+}
+```
+Keep an eye on your quota limits to maintain smooth operations!
+
+### New feature-flag for error subscription management
+We've introduced a new feature-flag `subscribe_to_error_by_default`, designed to streamline error subscription management within flows. This feature is accessible at both Tenant and Contract levels.
+By default, this feature is set to `true` for all tenants, ensuring seamless error tracking. You can customize this setting using API calls through `POST` and `PATCH` methods at the following endpoints:
+* v2/tenants/:id
+* v2/contracts/:id
+
+### Review platform compatibility with Kubernetes v1.27 and v1.28.
+The update of k8s to version 1.28 is planned for the 27th of April 2024.
+
+### New Dashboard metric: Messages in Queues
+Introducing a new metric to our Dashboard: Messages in Queues. This metric provides valuable insights into the status of unprocessed messages and offers a detailed list of flows along with their respective message queue counts.
+Now, you can easily monitor message queue performance, ensuring efficient message processing across your workflows.
+{% include img.html max-width="100%" url="/assets/img/RN/2417/Dashboard.png" title="Dashboard" %}
+
+### `Run Now` Button in Flow Designer Header
+The `Run Now` button is now available in the header of active flows, allowing users to effortlessly initiate the run-now functionality directly from the flow designer interface.
+{% include img.html max-width="100%" url="/assets/img/RN/2417/RunNow.png" title="Run Now" %}
+
+### New environment variable: `USER_NAME` in email template
+Added a new environment variable `USER_NAME` to the email template `workspace-invite-new-user`. This variable dynamically populates the receiver’s full name (`firstName` and `lastName`) in the invitation email.
+Now, the email notifications can be personalized with the recipient's name, creating a more engaging and tailored experience for new users joining your workspace.
+
+### Updated UI for Credentials section with checkboxes
+Updated UI for the credentials section, implementing a cleaner and more user-friendly design for checkboxes.
+
+## Fixed Bugs
+
+*   **FIXED** random plus and fork icons are displayed on the flow-designer page after adding a step in the middle of the flow
+*   **FIXED** cases where no new threads if the trigger hasn't emitted the messages since March 28th
```

Comment:
maybe we can leave here the link to the incedent descriptioin?

Reply:
Not sure. We don't have a published description. We have sent it via email.

Reply:
ok, let's remove the part after `since`. Because this bug was in the code from previous release and affected only westeros